README.md

```diff
@@ -1 +1,119 @@
-# Fast-DDS-statistics-backend
\ No newline at end of file
+# Fast DDS Statistics Backend
+
+<a href="http://www.eprosima.com"><img src="https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcSd0PDlVz1U_7MgdTe0FRIWD0Jc9_YH-gGi0ZpLkr-qgCI6ZEoJZ5GBqQ" align="left" hspace="8" vspace="2" width="100" height="100" ></a>
+
+*eProsima Fast DDS Statistics Backend* is a C++ library that provides collection and procession the statistics measurements reported by *Fast DDS Statistics Module*.
+The Backend provides a easy-to-use and user friendly API that abstracts the details of receiving and processing
+*Fast DDS* statistical data, so that applications can focus on providing great user experiences.
+*Fast DDS Statistics Backend* allows for:
+
+* Monitoring the various aspects reported by the Statistics Module for as many DDS Domains and *Fast DDS* Discovery Server network as desired.
+* Retrieve the DDS network graph.
+* Retrieve QoS settings about the DDS entities deployed in the network.
+* Receive notification on statistics updates for various events using a listener-callback mechanism.
+* Check whether an entity is active.
+* Retrieve statistics on all the aspects of the communication reported by the *Fast DDS Statistics Module*, with the possibility of specifying time windows, bins, and different statistics to apply.
+
+## Installation guide
+
+This tutorial shows how to build and install *Fast DDS Statistics Backend* using [colcon](https://colcon.readthedocs.io), a command line tool to build sets of software packages.
+To do so, `colcon` and `vcstool` need to be installed:
+
+```bash
+pip install -U colcon-common-extensions vcstool
+```
+
+### Dependencies
+
+*Fast DDS Statistics Backend* depends on [Fast DDS](https://github.com/eProsima/Fast-DDS) and [Fast CDR](https://github.com/eProsima/Fast-CDR).
+For simplicity, this tutorial will build these dependencies alongside the backend.
+More advanced users can build or link to this packages separately.
+Install *Fast DDS* dependencies running:
+
+```bash
+sudo apt update
+sudo apt install -y \
+    libasio-dev \
+    libtinyxml2-dev
+```
+
+### Build and install
+
```

Comment:
Later on a directory `<path_to_ws>` is specified, but there is no reference to it here. 
Maybe adding something like the following is enough:
```
# Navigate to the chosen workspace directory
cd <path_to_ws>
```